index.php

```diff
@@ -10,75 +10,104 @@
   <section id="posts">
 
 <?php
-if( have_posts() ) {
-  while( have_posts() ) {
-    the_post();
-    $author = get_post_meta($post->ID, '_igv_post_author', true);
-    $date = get_post_meta($post->ID, '_igv_post_date', true);
-    if (!empty($date)) {
-      $time = new \Moment\Moment(date('c', $date));
+if (!is_single()) {
+
+  $args = array (
+    'post_type'              => array( 'post', 'actividad' ),
+    'posts_per_page'         => '-1',
+  );
+
+  $query = new WP_Query( $args );
+
+} else {
+  $query = $wp_query;
+}
+
+if( $query->have_posts() ) {
+  while( $query->have_posts() ) {
+    $query->the_post();
+
+    if (get_post_type() == 'post') {
+      $author = get_post_meta($post->ID, '_igv_post_author', true);
+      $date = get_post_meta($post->ID, '_igv_post_date', true);
+      if (!empty($date)) {
+        $time = new \Moment\Moment(date('c', $date));
```

Comment:
you can append an @ to the unix timestamp that the datepicker saves to parse it into a Moment